### PR TITLE
[#174621326] API getUser for MyPortal

### PR DESCRIPTION
**Pull request by BurnedMarshal**
This PR depends on #703

### CODE REVIEW

**Review by gunzip — COMMENTED** [review]
lgtm to me apart minor changes

File: src/controllers/ssoController.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ */
+export const getUserForMyPortal = (
+  req: express.Request
+): Promise<
+  | IResponseErrorValidation
+  | IResponseErrorInternal
+  | IResponseSuccessJson<MyPortalUser>
+> =>
+  withUserFromRequest(req, async user =>
+    MyPortalUser.decode({
```
[the remark this answers is not in the document]
Comment (BurnedMarshal) [reply src/controllers/ssoController.ts]: yes, fixed in dcc22ef

File: src/controllers/ssoController.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/**
+ * This controller handles requests made from the PagoPA backend.
```
[the remark this answers is not in the document]
Comment (BurnedMarshal) [reply src/controllers/ssoController.ts]: fixed in dcc22ef

File: src/config.ts [hunk 3]
```diff
@@ -228,6 +228,18 @@ export const ALLOW_PAGOPA_IP_SOURCE_RANGE = decodeCIDRs(
   return process.exit(1);
 });
 
+// IP(s) or CIDR(s) allowed for payment manager endpoint
```
[the remark this answers is not in the document]
Comment (BurnedMarshal) [reply src/config.ts]: fixed in dcc22ef

**Comment by gunzip** [comment]
#703 is merged, can you rebase this against master and remove draft?

**Review by gunzip — APPROVED** [review]

**Pull request merged**